src/Model/Photo.php

```diff
@@ -369,28 +369,28 @@ public static function storeByData($uid, $imagedata = "", $url = "")
 		$logger = $a->getLogger();
 		$profiler = $a->getProfiler();
 
-		$r = DBA::p(
+		$isStored = DBA::p(
```

Comment:
Please replace this call by `User::getOwnerById`

Reply:
What about the additional `user.blocked = 0` ? This wouldn't be included